tests/e2e/specs/shopper/checkout-empty.test.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Internal dependencies
+ */
+import { shopper } from '../../../utils';
+
+if ( process.env.WOOCOMMERCE_BLOCKS_PHASE < 2 )
+	// eslint-disable-next-line jest/no-focused-tests
+	test.only( `skipping ${ block.name } tests`, () => {} );
```

Comment:
The PR looks great! The only thing I'd still change would be this part:

```suggestion
	test.only( `Skipping checkout tests`, () => {} );
```